TripScanner/src/main/resources/templates/sign.html

```diff
@@ -0,0 +1,105 @@
+    {{>header}}
+
+    <div class="container py-5 h-100">
+        <div class="row d-flex justify-content-center align-items-center h-100">
+
+            {{^correct}}
+            <div class="col-md-12">
+                <h3><span class="badge bg-danger">Password and repeated password fields were not equal</span></h3>
+            </div>
+            {{/correct}}
+
+            {{^noEmptyFields}}
+            <div class="col-md-12">
+                <h3><span class="badge bg-danger">Fields marked with "*" must be filled</span></h3>
+            </div>
+            {{/noEmptyFields}}
+
+            {{^goodEmail}}
+            <div class="col-md-12">
+                <h3><span class="badge bg-danger">Email was not valid</span></h3>
+            </div>
+            {{/goodEmail}}
+
+            <div class="col">
+                <div class="card card-registration my-4">
+                    <div class="row g-0" style="background-color: #A1C6EA">
+                        <div class="col-xl-6 d-none d-xl-block">
+                            <form action="/sign" method="post">
+                                <div class="photoCard">
+                                    <img src="/img/userPhoto.png" alt="Sample photo" class="img-fluid"
+                                         style="border-top-left-radius: .55rem; border-bottom-left-radius: .25rem;"
+                                         width="200px" />
+                                    <input type="file" id="file">
```

Comment:
Maybe you should add the accepted file extensions to prevent problems

Reply:
Photo upload is yet to be fully implemented